### ISSUE TITLE
Make _INFO files validation a configurable option

### ISSUE THREAD

**Issue by yruslan** [AbsaOSS/enceladus#1322]
## Background
Since strict validation of conformance rules (#1285) is a breaking change. To make the gradual transition it makes sense to make this option configurable and turned off by default.

## Feature
Make control measurements and checkpoint (_INFO files) validation a configurable option.
One option should apply for both Standardization and Conformance.

## Proposed Solution [Optional]
`control.info.validation=false`

**Comment by benedeki** [comment]
I would definitely make it `on` by default. That's the correct behavior.
But I am still not sure it's a good approach in general. It will unblock the release deployment for sure. But we just postpone the validation and will always face the sitaution of bad INFO files.

Perhaps a three value setting would be a solution (keywords just suggestions)?
`strict`, `warning` and `off`

**Comment by AdrianOlosutean** [comment]
The default could be warning?

**Comment by benedeki** [comment]
Considering, that would be my preferred solution. 🤔

**Comment by AdrianOlosutean** [comment]
So what should exactly happen on warning, jut logging that?

**Comment by Zejnilovic** [comment]
Just log something like:
WARN: _INFO file does not have all the checkpoints for full data quality compliance.

**Comment by yruslan** [comment]
Maybe the same message as if it is a validation error, just `log.warn` instead of `throw`?

Warning messages could benefit the verbosity you are adding to errors.

**Comment by AdrianOlosutean** [comment]
Release notes:
Configuration for _INFO file validation type
